### PR TITLE
Fix error for detecting internal images properly

### PR DESCRIPTION
**Pull request by TrentHouliston**
Internal images are no longer working since `repository` was changed to `image` to allow for caching

### CODE REVIEW

**Review by JosephusPaye — APPROVED** [review]
LGTM, assuming it works

File: tools/utility/dockerise/defaults.py [hunk 1]
```diff
@@ -7,6 +7,6 @@
 directory = "NUbots"
 
 
-def is_internal_image(image):
-    split = image.split(":")
-    return len(split) == 2 and split[0] == repository and (split[1] in platform.list() or split[1] == "selected")
+def is_internal_image(tag):
+    split = tag.split(":")
+    return len(split) == 2 and split[0] == image and (split[1] in platform.list() or split[1] == "selected")
```
Comment (Bidski) [inline tools/utility/dockerise/defaults.py:12]: Surely `repository` is more semantically correct here?

Comment (TrentHouliston) [reply tools/utility/dockerise/defaults.py:12]: Nah, in docker you have things split up into 3
`repository/image:thing`
We're checking here that `image` is `nubots` (local image)

Comment (Bidski) [reply tools/utility/dockerise/defaults.py:12]: Fair enough, I dont see any issues then

**Pull request merged**